fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -448,7 +448,7 @@ void FixedLagSmoother::transactionCallback(
       pending_transactions_.end(),
       transaction->stamp(),
       comparator);
-    pending_transactions_.insert(position, {sensor_name, std::move(transaction)});  // NOLINT
+    position = pending_transactions_.insert(position, {sensor_name, std::move(transaction)});  // NOLINT
```

Comment:
This is required because `position` could be `end()` before `insert`ing.

Reply:
:+1: